### ISSUE TITLE
playerlogger.js blocked by adblockers

### ISSUE THREAD

**Issue by adamk33n3r** [ForgeEssentials/FERemotePanel#6]
If you look here https://easylist-downloads.adblockplus.org/easyprivacy.txt for "playerlogger" you'll see an entry.

I feel like a lot of people (myself included) use adblockers so it might be wise to change the name of this file.

Thoughts?

**Comment by yuuka-miya** [comment]
Remote panel is run on your own server - perhaps you could make an adblock exception?

@olee might want to look into this though.

**Comment by adamk33n3r** [comment]
Right. Of course I can make an exception. It's just not immediately obvious why the site doesn't load.

**Comment by olee** [comment]
This is weird. 
I also use adblock and it loaded just fine for me...
Will need to check this sometime.
